### PR TITLE
cleaning: removing util module from builtin-tasks

### PR DESCRIPTION
**Pull request by robert-zaremba**
https://www.pivotaltracker.com/n/projects/2452320#

### CODE REVIEW

**Comment by robert-zaremba** [comment]
Few notes: 
+ For imports with types only we should use `import type ...`
+ we should prefer `interface A {... }`  over `type A = { ... }`

File: packages/builder/test/builtin-tasks/deploy.ts [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+import { TASK_DEPLOY } from "../../src/builtin-tasks/task-names";
+import { loadFilenames } from "../../src/builtin-tasks/deploy";
 
 describe("Deploy task", function () {
   useCleanFixtureProject("scripts-dir")
 
+  it("Should load scripts in the right order", function(){
+    const ls = loadFilenames("scripts");
+    let e = ls[0];
+    for (let i = 1; i<ls.length; ++i) {
```
Comment (Invertisment) [inline packages/builder/test/builtin-tasks/deploy.ts:16]: Wasn't it enought to run `assert.deepEquals`?

Comment (robert-zaremba) [reply packages/builder/test/builtin-tasks/deploy.ts:16]: Amount of lines would be similar, because I would need to copy, sort and then do deep comparison

**Comment by robert-zaremba** [comment]
Thanks for review

**Pull request merged**